lib/simple-dom/document/node.js

```diff
@@ -163,4 +162,21 @@ ChildNodes.prototype.item = function(index) {
   return child;
 };
 
+if (Object.defineProperty) {
```

Comment:
Would you rather an extension or a smarter implementation which caches the result if no operations are taken on the children (removeChild, appendChild, etc)?